website/cue/reference/components/sinks.cue

```diff
@@ -163,6 +163,10 @@ components: sinks: [Name=string]: {
 								required:    true
 								type: string: {
 									examples: features.send.encoding.codec.enum
+									enum: {
+										json: "Array of JSON objects, which must be a JSON array containing JSON objects."
+										text: "Newline-delimited text, with each line forming a message."
+										}
```

Comment:
you might be missing few here, https://github.com/timberio/vector/issues/8530#issuecomment-916947401 mentions also `ndjson` and `logfmt`.

Reply:
plz note that I've never done any Cue, so I honestly don't (~have any clue~) understand how those values are interpolated around the code. I would imagine that there's some some magic that filters them over the list in the actual component (https://github.com/timberio/vector/blob/6c94aad0f55b8e50b624f50c9d656f26637c2bb9/website/cue/reference/components/sinks/aws_s3.cue#L33-L36).

Reply:
@dre-sctg Thanks - it looks like this components/sinks.cue file would be where the generic usage would go, i.e. set enum to `features.send.encoding.codec`, which would be set in the appropriate `.cue` file: website/cue/reference/components/sinks/aws_kinesis_firehose.cue, for example (which lists enum options as json and text) or website/cue/referrence/components/sinks/aws_s3.cue (which lists enum options as ndjson or text).

Will need to update the layouts/partials/data.html file to update the HTML rendering of the enum options. Lots of files linked it seems 🤔